### ISSUE TITLE
Notice: Uninitialized string offset: 0

### ISSUE THREAD

**Issue by narendravaghela** [cakephp/cakephp-codesniffer#170]
I got below error while running phpcs:

Notice: Uninitialized string offset: 0 in \vendor\cakephp\cakephp-codesniffer\CakePHP\Sniffs\Commenting\FunctionCommentSniff.php on line 258

The code on line 258 is as below:

```php
// Starts with a capital letter and ends with a fullstop.
$firstChar = $comment{0};
```

It should first check if there is a character at position 0.

**Comment by narendravaghela** [comment]
As a workaround, I have modified that line like,

`$firstChar = isset($comment{0}) ? $comment{0} : '';`

Please let me know if it is correct way or not? If yes, I can open a pull request.

**Comment by dereuromark** [comment]
Looks OK to me, a PR is welcome.

**Comment by narendravaghela** [comment]
PR is created.
